ansible/roles/atmo-realvncserver/files/config.custom

```diff
@@ -1,6 +1,8 @@
 # vncserver common.custom configuration.
 SecurityTypes=RA2
-Permissions=root:f,%core-services:f,:f
+#TODO: Line below CyVerse ONLY
```

Comment:
Perhaps make this file a template and allow the consumer to define this as a variable?